src/ConnectedMode/UI/ManageBinding/ManageBindingDialog.xaml

```diff
@@ -90,7 +90,7 @@
                         </Grid.ColumnDefinitions>
 
                         <Button Grid.Column="0" Content="{x:Static res:UiResources.UnbindButton}" IsEnabled="{Binding Path=IsUnbindButtonEnabled}" 
```

Comment:
Discussed on slack: the button is a bit hard to see in the dark theme (same as the shared binding button)